src/c++/perf_analyzer/docs/benchmarking.md

```diff
@@ -30,47 +32,85 @@ This is the default mode for Perf Analyzer.
+arguments. By default, Perf Analyzer expects the Triton instance to already be
+running. You can configure C API mode using the
+[`--service-kind`](cli.md#--service-kindtritontriton_c_apitfservingtorchserve)
+option. In additon, you will need to point Perf Analyzer to the Triton server
+library path using the
+[`--triton-server-directory`](cli.md#--triton-server-directorypath) option and
+the model repository path using the
+[`--model-repository`](cli.md#--model-repositorypath) option.
 
-Use the --help option to see complete list of supported command line arguments.
-By default perf_analyzer expects the Triton instance to already be running. You can configure the C API mode using the [`--service-kind`](cli.md#--service-kindtritontriton_c_apitfservingtorchserve) option. In additon, you will need to point
-perf_analyzer to the Triton server library path using the [`--triton-server-directory`](cli.md#--triton-server-directorypath) option and the model
-repository path using the [`--model-repository`](cli.md#--model-repositorypath) option.
-If the server is run successfully, there is a prompt: "server is alive!" and perf_analyzer will print the stats, as normal.
 An example run would look like:
 
 ```
-perf_analyzer -m graphdef_int32_int32_int32 --service-kind=triton_c_api --triton-server-directory=/opt/tritonserver --model-repository=/workspace/qa/L0_perf_analyzer_capi/models
+$ perf_analyzer -m my_model --service-kind=triton_c_api --triton-server-directory=/opt/tritonserver --model-repository=/my_model_repository
+...
+*** Measurement Settings ***
+  Service Kind: Triton C-API
+  Using "time_windows" mode for stabilization
+  Measurement window: 5000 msec
+  Using synchronous calls for inference
+  Stabilizing using average latency
+
+Request concurrency: 1
+  Client: 
+    Request count: 353
+    Throughput: 19.6095 infer/sec
+    Avg latency: 50951 usec (standard deviation 2265 usec)
+    p50 latency: 50833 usec
+    p90 latency: 50923 usec
+    p95 latency: 50940 usec
+    p99 latency: 50985 usec
+    
+  Server: 
+    Inference count: 353
+    Execution count: 353
+    Successful request count: 353
+    Avg request latency: 50841 usec (overhead 20 usec + queue 63 usec + compute input 35 usec + compute infer 50663 usec + compute output 59 usec)
+
+Inferences/Second vs. Client Average Batch Latency
+Concurrency: 1, throughput: 19.6095 infer/sec, latency 50951 usec
 ```
 
 ## Non-supported functionalities
 
-There are a few functionalities that are missing from the C API. They are:
+There are a few functionalities that are missing from C API mode. They are:
 
-1. Async mode (`-a`)
-2. Using shared memory mode ([`--shared-memory=cuda`](cli.md#--shared-memorynonesystemcuda) or `--shared-memory=system`)
-3. Request rate range mode
+1. Async mode ([`--async`](cli.md#--async))
+2. Shared memory mode
+   ([`--shared-memory=cuda`](cli.md#--shared-memorynonesystemcuda) or
+   [`--shared-memory=system`](cli.md#--shared-memorynonesystemcuda))
```

Comment:
We also have testing for these in C-API now so we can remove this bullet point too.

Reply:
Removed 👍